doc/api/tty.md

```diff
@@ -63,7 +63,7 @@ added: v0.7.7
 * `mode` {boolean} If `true`, configures the `tty.ReadStream` to operate as a
   raw device. If `false`, configures the `tty.ReadStream` to operate in its
   default mode. The `readStream.isRaw` property will be set to the resulting
-  mode.
+  mode. Returns `this` - the read stream instance.
```

Comment:
Needs to be on a separate line:
```md
* Returns: {this} - the read stream instance.
```

Reply:
fml lol

Reply:
so it needs to be {this} not `this`, is that right?

Reply:
yes correct.

Reply:
please check the new commit to see if it's right, thanks

Reply:
It should be what @vsemozhetbyt put in his comment, verbatim :)